app/views/comfy/blog/posts/index.html.haml

```diff
@@ -4,11 +4,16 @@
       %h1 Blog Posts
 
       - @blog_posts.each do |post|
-
-        %h2
-          = link_to post.title, comfy_blog_post_path(@cms_site.path, post.year, post.month, post.slug)
-        .date
-          = post.published_at.to_s(:db)
+        .row
+          - image_attached = post.file.present? && post.file.attachment.attached? && post.file.attachment.content_type.include?("image")
```

Comment:
Should probably be a model method like `has_image?`, but I felt I'd leave that decision up to you.